README.md

```diff
@@ -2,16 +2,16 @@
  This project uses PhantomJs [http://phantomjs.org] executables to generated pdf from html strings.
  This library should work on Windows, Linux and OSX, it checks for the platform on the fly and uses the appropriate executable.
 
-# Example
+## Example
   For a simple example have a look at Example.cs
```

Comment:
@JoaoLucasGtr Lets chage this paragraph. Instead of Example.cs we can either embed the whole code in Source/Sandbox/Program.cs or just mention the path here.

And I think the bullet point below ``` Do not forget to include... ``` is no longer needed because you went around that.

Reply:
Agreed.